.github/workflows/release.yaml

```diff
@@ -324,8 +324,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
+          ref: "main"
```

Comment:
This ensures we bump examples and regenerate docs in main always.

Reply:
This workflow can run from another branch, do we want to filter this to main? Not completely aware of this workflow yet.